src/machinetalk/webtalk/webtalk_wsproxy.cc

```diff
@@ -462,9 +462,9 @@ callback_http(struct libwebsocket_context *context,
 	    zloop_poller_end (self->netopts.z_loop, &wss->wsqin_pollitem);
 
 	    if (wss->socket != NULL)
-		zsocket_destroy (self->netopts.z_context, wss->socket);
-	    zsocket_destroy (self->netopts.z_context, wss->wsq_in);
-	    zsocket_destroy (self->netopts.z_context, wss->wsq_out);
+		zsock_destroy (self->netopts.z_context, wss->socket);
+	    zsock_destroy (self->netopts.z_context, wss->wsq_in);
+	    zsock_destroy (self->netopts.z_context, wss->wsq_out);
```

Comment:
zsock_destroy() just takes  one arg
`CZMQ_EXPORT void  zsock_destroy (zsock_t **self_p);`

Does this mean you have to call `zsock_disconnect` first, or will destroy do that ?